lib/rs-rails-base/features/facebook.rb

```diff
@@ -37,6 +39,9 @@ def self.inject_files(templates_path)
         inject_into_file('spec/routing/sessions_routing_spec.rb',
                          "sessions#create'\)\n    end",
                          "#{templates_path}sessions_routing_spec.rb")
+        inject_into_file('config/initializers/figaro.rb',
+                         'variables += %w[SECRET_KEY_BASE]',
```

Comment:
In the other repo we are not requiring this variable if `Rails.env.test?`. Is this behaving the same way?